### ISSUE TITLE
Exception ignored in

### ISSUE THREAD

**Issue by ctlaltdefeat** [aio-libs/aiohttp#315]
Hello,

When running the following code with Python 3.4.3:

``` python
import asyncio, aiohttp
conn = aiohttp.TCPConnector(verify_ssl=False, share_cookies=True)

def main_loop():
    r = yield from aiohttp.request('get', 'http://www.google.com', connector=conn)
    print((yield from r.text()))


if __name__ == '__main__':
    loop = asyncio.get_event_loop()
    loop.run_until_complete(main_loop())
```

I indeed get the desired output, however I also get `Exception ignored in: Exception ignored in:`. 
I've tried to try-except in various places but haven't been able to understand what's causing it. 
I've noticed that if I don't use the explicit connector that I defined, I only get  `Exception ignored in:` once.

**Comment by fafhrd91** [comment]
could you post logs?

**Comment by ludovic-gasc** [comment]
You can enable debug logging globally to add `logging.basicConfig(level=logging.DEBUG)` at the beginning of the script to have more details.

You should also set the environment variable PYTHONASYNCIODEBUG to 1: https://docs.python.org/3/library/asyncio-dev.html

**Comment by ctlaltdefeat** [comment]
Closing the connection at the end of main_loop and closing main_loop as the end of main solved it. I guess this can be closed :)

**Comment by kxepal** [comment]
Reproduced in aiocouchdb test suite:

```
DEBUG:asyncio:Close <_UnixSelectorEventLoop running=False closed=False debug=True>
ERROR:asyncio:Task was destroyed but it is pending!
source_traceback: Object created at (most recent call last):
  File "/home/kxepal/projects/aiocouchdb/venv/bin/nosetests", line 11, in <module>
    sys.exit(run_exit())
  File "/home/kxepal/projects/aiocouchdb/venv/lib64/python3.4/site-packages/nose/core.py", line 121, in __init__
    **extra_args)
  File "/usr/lib64/python3.4/unittest/main.py", line 93, in __init__
    self.runTests()
  File "/home/kxepal/projects/aiocouchdb/venv/lib64/python3.4/site-packages/nose/core.py", line 207, in runTests
    result = self.testRunner.run(self.test)
  File "/home/kxepal/projects/aiocouchdb/venv/lib64/python3.4/site-packages/nose/core.py", line 62, in run
    test(result)
  File "/home/kxepal/projects/aiocouchdb/venv/lib64/python3.4/site-packages/nose/suite.py", line 178, in __call__
    return self.run(*arg, **kw)
  File "/home/kxepal/projects/aiocouchdb/venv/lib64/python3.4/site-packages/nose/suite.py", line 225, in run
    test(orig)
  File "/home/kxepal/projects/aiocouchdb/venv/lib64/python3.4/site-packages/nose/suite.py", line 178, in __call__
    return self.run(*arg, **kw)
  File "/home/kxepal/projects/aiocouchdb/venv/lib64/python3.4/site-packages/nose/suite.py", line 225, in run
    test(orig)
  File "/home/kxepal/projects/aiocouchdb/venv/lib64/python3.4/site-packages/nose/suite.py", line 178, in __call__
    return self.run(*arg, **kw)
  File "/home/kxepal/projects/aiocouchdb/venv/lib64/python3.4/site-packages/nose/suite.py", line 225, in run
    test(orig)
  File "/home/kxepal/projects/aiocouchdb/venv/lib64/python3.4/site-packages/nose/case.py", line 46, in __call__
    return self.run(*arg, **kwarg)
  File "/home/kxepal/projects/aiocouchdb/venv/lib64/python3.4/site-packages/nose/case.py", line 134, in run
    self.runTest(result)
  File "/home/kxepal/projects/aiocouchdb/venv/lib64/python3.4/site-packages/nose/case.py", line 152, in runTest
    test(result)
  File "/usr/lib64/python3.4/unittest/case.py", line 625, in __call__
    return self.run(*args, **kwds)
  File "/usr/lib64/python3.4/unittest/case.py", line 577, in run
    testMethod()
  File "/home/kxepal/projects/aiocouchdb/aiocouchdb/tests/utils.py", line 37, in wrapper
    return testcase.loop.run_until_complete(future)
  File "/usr/lib64/python3.4/asyncio/base_events.py", line 304, in run_until_complete
    self.run_forever()
  File "/usr/lib64/python3.4/asyncio/base_events.py", line 276, in run_forever
    self._run_once()
  File "/usr/lib64/python3.4/asyncio/base_events.py", line 1164, in _run_once
    handle._run()
  File "/usr/lib64/python3.4/asyncio/events.py", line 120, in _run
    self._callback(*self._args)
  File "/usr/lib64/python3.4/asyncio/tasks.py", line 292, in _wakeup
    self._step(value, None)
  File "/usr/lib64/python3.4/asyncio/tasks.py", line 236, in _step
    result = coro.send(value)
  File "/usr/lib64/python3.4/asyncio/coroutines.py", line 95, in send
    return self.gen.send(value)
  File "/home/kxepal/projects/aiocouchdb/aiocouchdb/v1/tests/test_database.py", line 186, in test_changes_longpoll
    result = yield from self.db.changes(feed='longpoll')
  File "/usr/lib64/python3.4/asyncio/coroutines.py", line 95, in send
    return self.gen.send(value)
  File "/home/kxepal/projects/aiocouchdb/aiocouchdb/v1/database.py", line 381, in changes
    return LongPollChangesFeed(resp, buffer_size=feed_buffer_size)
  File "/home/kxepal/projects/aiocouchdb/aiocouchdb/feeds.py", line 47, in __init__
    asyncio.Task(self._loop(), loop=loop)
task: <Task pending coro=<Feed._loop() running at /home/kxepal/projects/aiocouchdb/aiocouchdb/feeds.py:53> wait_for=<Future finished result=True created at /home/kxepal/projects/aiohttp/aiohttp/streams.py:319> created at /home/kxepal/projects/aiocouchdb/aiocouchdb/feeds.py:47>
```

TL;DR it happens because I have an active task in background for non closed connection when test suite calls tearDown and closes the loop. I think it's good now that such exception is raised since I could make a little cleanup now.
